code/game/machinery/computer/apc_control.dm

```diff
@@ -130,7 +124,7 @@
 			active_apc = null
 		to_chat(usr, "<span class='robot notice'>[icon2html(src, usr)] Connected to APC in [APC.area]. Interface request sent.</span>")
 		log_activity("remotely accessed APC in [APC.area]")
-		APC.interact(usr, GLOB.not_incapacitated_state)
+		APC.ui_interact(usr, state = GLOB.not_incapacitated_state)
```

Comment:
user*

Reply:
Gotcha will do that on Sunday

Reply:
`usr` isn't used anywhere else in this topic()

Reply:
True but I can probably convert it over.

Reply:
no this is my bad, i didn't realize the proc changed